examples/react/field-errors-from-form-validators/src/index.tsx

```diff
@@ -0,0 +1,124 @@
+import { useForm } from '@tanstack/react-form'
+import * as React from 'react'
+import { createRoot } from 'react-dom/client'
+
+async function verifyAgeOnServer(age: number) {
+  return age >= 13
+}
+
+async function checkIfUsernameIsTaken(name: string) {
+  const usernames = ['user-1', 'user-2', 'user-3']
+  return usernames.includes(name)
+}
+
+export default function App() {
+  const form = useForm({
+    defaultValues: {
+      username: '',
+      age: 0,
+    },
+    validators: {
+      onSubmitAsync: async ({ value }) => {
+        console.log({ value })
+        // Verify the age on the server
+        const isNotOlderThan13 = !(await verifyAgeOnServer(value.age))
+
+        // Verify the availability of the username on the server
+        const isUsernameTaken = await checkIfUsernameIsTaken(value.username)
```

Comment:
Personal preference: I'd keep both these positive here: `verifyAgeOnServer` is returning true if the value is ok, `checkIfUsernameIsTaken` is returning true if the value is *not* ok.

Let's make them consistent, an example:
```ts
async function verifyAgeOnServer(age: number) {
  return age < 13
}
...
const isTooYoung = await verifyAgeOnServer(value.age)
```

Reply:
Great points, I fixed it!